wagn/lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -126,6 +125,7 @@ def config
 
     inside "config" do
       template "application.rb"
+      template 'routes.erb', "routes.rb"
```

Comment:
are you proposing to make it so that every deck now has to have its own routes file?  Does this mean existing wagns won't work without one?  I would prefer to make it so that sites can work without their own routes.rb file (and can have one if they want to customize); is that workable?